src/lib/components/customId.svelte

```diff
@@ -29,7 +29,11 @@
     </svelte:fragment>
     <svelte:fragment slot="content">
         <div class="form">
-            <InputId bind:value={id} {autofocus} />
+            {#if name === 'Project'}
```

Comment:
I think we should add a new exported variable called

`export let isProject = false;`

Becuase we will introduce i18n at some point and this doesnt scale 👍🏻

Reply:
Thank you for letting me know, and I have made the changes